lib/common.js

```diff
@@ -410,11 +419,11 @@ async function check_filter_attachments(pg_pool, app_uuid) {
   }));
 }
 
-async function check_deployment_filters(pg_pool, app_uuid, dyno_type) {
+async function check_deployment_filters(pg_pool, app_uuid, dyno_type, features) {
```

Comment:
Codacy found an issue: [Async function 'check_deployment_filters' has a complexity of 10. Maximum allowed is 8.](https://app.codacy.com/manual/Akkeris/controller-api/pullRequest?prid=5835669)